VAMobile/src/api/claimsAndAppeals/uploadFileToClaim.tsx

```diff
@@ -32,8 +32,8 @@ const uploadFileToClaim = ({ claimID, request, files }: UploadFileToClaimParamat
     const payload = JSON.parse(
       JSON.stringify({
         files: fileStrings,
-        trackedItemId: request.trackedItemId,
-        document_type: request.documentType,
+        trackedItemId: request?.trackedItemId,
```

Comment:
how do we confirm that trackedItemID can be nil on the backend? Did they make this change and is it associated to a ticket?